spec/features/idv/liveness/upgrade_to_strong_ial2_spec.rb

```diff
@@ -49,6 +59,16 @@
       click_agree_and_continue_optional
 
       expect(page.current_path).to eq(idv_doc_auth_welcome_step)
+
+      complete_all_doc_auth_steps
+      click_continue
+      fill_in 'Password', with: user.password
+      click_continue
+      click_acknowledge_personal_key
+      click_agree_and_continue
+
+      expect(current_url).to start_with('http://localhost:7654/auth/result')
+      expect(user.active_profile.includes_liveness_check?).to be_truthy
```

Comment:
Just to clarify, this is not what I meant --- I was talking specifically about the claims in the ID token that API consumers receive --- if I am an agency and I request strict IAL 2, I want to see something in the response that says "this is indeed strict IAL 2", other agencies don't have the ability to look into the database like this